Makefile

```diff
@@ -19,7 +19,13 @@ run: all
 build-sass: $(SASS_SRC)
 	$(SASS_COMPILER) $(MAIN_FILE) $(OUTPUT_FILE)
 
+build-docker-image:
+	docker build -t 2016-site .
+
+docker:
```

Comment:
Fair enough, but, it does? make should be able to tell when the file gets modified...